TechnologyAssessmentApp/TechnologyAssessmentApp.xcodeproj/project.pbxproj

```diff
@@ -699,12 +699,13 @@
 			buildSettings = {
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
 				CLANG_ENABLE_MODULES = YES;
-				CODE_SIGN_STYLE = Automatic;
-				DEVELOPMENT_TEAM = S6BSNZ5TEB;
+				CODE_SIGN_STYLE = Manual;
```

Comment:
Any reason why you are switching to manual ? Better doing automatic to avoid signing issues for teammates.

Reply:
Yes I also prefer Automatic.As For now, there were no provisioning profile, therefor i kept it to manual.

Reply:
If you keep it to manual, it would be more difficult for other developers to clone and run the build. Better switch it to automatic to avoid signing problems. Also add on the README they need to change their team to be able to build.